### PR TITLE
Extract Auth service

### PR DESCRIPTION
**Pull request by donesvad**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -29,11 +29,8 @@ logging.logback.rollingpolicy.clean-history-on-start=false
 spring.security.user.name=endpoint_admin
 spring.security.user.password=Z%zk^9VN)^D@UWzb
 
+# TODO: CHANGE TO ENV VARIABLE AND PASS IT AS A SECRET PARAMETER
```
Comment (alexandrter) [inline src/main/resources/application.properties]: You can already create a secret variable in github and add it like we did with datasouce properties. I don't see why it has to be done in a separate commit

**Review by alexandrter — APPROVED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (donesvad) [reply src/main/resources/application.properties]: Done, thx
